scripts/f5-cookie-decode.nse

```diff
@@ -0,0 +1,70 @@
+-- See here: https://support.f5.com/csp/article/K6917
+description = [[
+Decodes any unencrypted F5 BIG-IP cookies in the HTTP response.
```

Comment:
Can you add a little more documentation about this? Maybe just add a reference link.